### PR TITLE
feat(dora): e2e

### PR DESCRIPTION
**Pull request by warren830**
# Summary

add e2e for change_lead_time_calculator.go and issue_deploy_connector.go
modify e2e for cicd_task_env_enricher.go
fix minor bugs in dora and github

### Does this close any open issues?
relate to #2853

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

**Review by abeizn — APPROVED** [review]
lgtm

File: plugins/github/tasks/commit_convertor.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ import (
 var ConvertCommitsMeta = core.SubTaskMeta{
 	Name:             "convertCommits",
 	EntryPoint:       ConvertCommits,
-	EnabledByDefault: false,
+	EnabledByDefault: true,
```
Comment (keon94) [inline plugins/github/tasks/commit_convertor.go:36]: @warren830 this has broken the e2e tests

**Comment by klesh** [comment]
Hey, guys, why this PR got merged while the tests failed?

File: plugins/github/tasks/commit_convertor.go (hunk 1 above)
Comment (warren830) [reply plugins/github/tasks/commit_convertor.go:36]: > 

This is fixed in other prs

**Pull request merged**